layouts/partials/sidebar/auto-collapsible-menu.html

```diff
@@ -1,26 +1,27 @@
 <!-- Auto collapsible section menu -->
 <ul class="list-unstyled collapsible-sidebar">
-  {{ $currentPage := . -}}
   <li class="mb-1">
-    <button data-link="home" class="btn btn-toggle align-items-centers">
+    <button class="static-page btn btn-toggle align-items-centers">
       <a class="docs-link rounded {{ if .IsHome }}active{{ end }}" href="{{ .Site.Home.Permalink }}">Home</a></li>
     </button>
   </li>
-  {{ if .Site.Params.options.changelog}}
-  <li class="mb-1">
-    {{ $active := in $currentPage.RelPermalink "changelog" }}
-    <button data-link="home" class="btn btn-toggle align-items-centers">
-      <a class="docs-link rounded {{ if $active }}active{{ end }}" href="{{ .Site.Home.Permalink }}/changelog">Changelog</a></li>
-    </button>
-  </li>
-  {{end}}
```

Comment:
I removed this one because in cohtml and UE4 the changelog is a regular static page that is added in the sidebar and in GameUIComponents we don't have changelog.